setup.py

```diff
@@ -1,15 +1,35 @@
-from cx_Freeze import setup, Executable
+from setuptools import setup, find_packages
+from io import open
+import qtsass
 
-build_exe_options = {
-    'include_msvcr': True,
-    'bin_includes': ['msvcp120.dll'],  # Force msvcp120 to be included (libsass dependency, not copied by include_mscvr)
-    'optimize': 2
-}
 
 setup(
-    name="qtsass",
-    description="Compile a SCSS file to a valid Qt CSS.",
-    executables=[Executable("qtsass/qtsass.py")],
-    options={'build_exe': build_exe_options},
-    requires=['libsass', 'cx_Freeze', 'watchdog'],
+    name=qtsass.__title__,
+    version=qtsass.__version__,
+    description=qtsass.__description__,
+    long_description=open("README.md", "r", encoding="utf-8").read(),
```

Comment:
We can clean this up a bit by moving out to a helper function.

https://github.com/spyder-ide/loghub/blob/master/setup.py#L34